crates/biome_cli/src/commands/mod.rs

```diff
@@ -77,11 +77,24 @@ pub enum BiomeCommand {
     #[bpaf(command)]
     Check {
         /// Apply safe fixes, formatting and import sorting
-        #[bpaf(long("apply"), switch)]
+        #[bpaf(long("apply"), switch, hide_usage)]
         apply: bool,
         /// Apply safe fixes and unsafe fixes, formatting and import sorting
-        #[bpaf(long("apply-unsafe"), switch)]
+        #[bpaf(long("apply-unsafe"), switch, hide_usage)]
         apply_unsafe: bool,
+
+        /// Fixes lint errors safely, formatting and import sorting
+        #[bpaf(long("fix"), switch)]
+        fix: bool,
+
+        /// Alias for `--fix`, fixes lint errors safely, formatting and import sorting
+        #[bpaf(long("write"), switch, hide_usage)]
+        write: bool,
```

Comment:
Could we display `--fix` and `--write` together as aliases? Not sure if bpaf supports that.

Otherwise, I could choose to hide `--fix` because `--write` is the one we have on every command, including `migrate` commands.

Reply:
We can try to use two `long` attributes.

Reply:
I just tried to pass two `long' attributes to the derive of the `write' field, but it didn't show the alias.

It's related to this discussion: https://github.com/pacak/bpaf/discussions/358